pywinauto/controls/hwndwrapper.py

```diff
@@ -1375,8 +1375,14 @@ def scroll(self, direction, amount, count = 1, retry_interval = None):
             message = win32defines.WM_VSCROLL
 
         # the constant that matches direction, and how much
-        scroll_type = \
-            self._scroll_types[direction.lower()][amount.lower()]
+        try:
+            scroll_type = \
+                self._scroll_types[direction.lower()][amount.lower()]
+        except KeyError:
+            raise ValueError("""Wrong arguments:
+                direction can be any of "up", "down", "left", "right"
+                amount can be any of "line", "page", "end"
```

Comment:
`"end"` is not possible here

Reply:
As I can see ```_scroll_types```dictionary does have "end"

Reply:
Maybe it's incomplete push? I've double checked current PR diff and there is no `"end"` key, only `"line"` and `"page"` for every scroll direction.

Reply:
I think, there is a bit of confusion here. There is an old, ```Hwndwrapper._scroll_types``` dictionary that does have "end", but ```UIAWrapper._scroll_types``` has been added only in this PR and it indeed doesn't have "end". This is because, win32 back end supports more options than UIA.

Reply:
Ah, got it, thanks!